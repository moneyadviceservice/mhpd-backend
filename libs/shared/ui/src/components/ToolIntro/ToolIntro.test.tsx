import React from 'react';
import { render, screen } from '@testing-library/react';
import { ToolIntro } from '.';

describe('ToolIntro component', () => {
  it('renders correctly ', () => {
    render(
      <ToolIntro>
        <p>Lorem ipsum</p>
      </ToolIntro>,
    );
    const toolIntro = screen.getByTestId('tool-intro');
    expect(toolIntro).toMatchSnapshot();
  });
});
