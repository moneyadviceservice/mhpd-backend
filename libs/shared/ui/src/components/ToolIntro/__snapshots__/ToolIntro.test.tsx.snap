// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToolIntro component renders correctly  1`] = `
<div
  class="t-tool-intro border-l-[6px] border-teal-300 pl-6 text-lg text-gray-800 font-normal leading-[26px]"
  data-testid="tool-intro"
>
  <p>
    Lorem ipsum
  </p>
</div>
`;
